src/models/Game.js

```diff
@@ -0,0 +1,53 @@
+import readlineSync from "readline-sync";
+
+import Race from "./Race";
+import { ERROR_MESSAGE, NUMBER_OF_MATCHES, ZERO } from "../constants";
+
+export class Game {
+  totalMatches;
+  race;
+  constructor() {
+    this.totalMatches = NUMBER_OF_MATCHES;
+    this.race = new Race();
+  }
+
+  start() {
+    const names = this.getNames();
+    if (names.length === 0) {
+      throw new Error(ERROR_MESSAGE.LACK_OF_MINIMUM_CARS);
+    }
+    this.race.setCars(...names);
+
+    Array.from({ length: this.totalMatches }, () => this.race.startRound());
+
+    this.printWinners();
+  }
+
+  getNames() {
+    const names = readlineSync
+      .question(
+        "경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분)."
+      )
+      .split(",");
+    return names;
+  }
```

Comment:
메서드 네이밍만으로 충분해보이네요 💯 

```suggestion
  getNames() {
    return readlineSync
      .question(
        "경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분)."
      )
      .split(",");
  }
```